### DISCUSSION TITLE
Headless Sunmao

### DISCUSSION THREAD

**Discussion by Yuyz0112** [smartxworks/sunmao-ui#600] [Ideas]
Headless Sunmao is an idea to build Web UI in backend runtime by using non-javascript programming languages.

You can watch a demo video and get more details on this [blog](http://www.myriptide.com/painless-web-ui-in-go/).

**Comment by ManmohanSarwara** [comment]
This is amazing work @Yuyz0112, Keep developing this... I also want to be part of this development.... Nice work!!!
